integration/tests/cook/test_basic.py

```diff
@@ -892,45 +921,46 @@ def jobs_query():
 
             util.wait_until(jobs_query, util.all_instances_done)
             # retry all jobs in the group
-            util.retry_jobs(self.cook_url, retries=2, groups=[group_uuid])
+            util.retry_jobs(self.cook_url, retries=2, groups=[group_uuid], **extra_retry_args)
             # wait for some job to start
             util.wait_until(group_query, util.group_some_job_started)
             # ensure none of the jobs are still in a failed state
             job_data = util.query_jobs(self.cook_url, job=jobs)
             self.assertEqual(200, job_data.status_code)
             for job in job_data.json():
-                job_details = f'Job details: {json.dumps(job, sort_keys=True)}'
+                job_details = f'failed_only={failed_only}; Job details: {json.dumps(job, sort_keys=True)}'
                 self.assertNotEqual('failed', job['state'], job_details)
         finally:
             # ensure that we don't leave a bunch of jobs running/waiting
             util.kill_groups(self.cook_url, [group_uuid])
 
+    def test_group_change_killed_retries(self):
+        self._help_test_group_change_killed_retries(failed_only=True)
```

Comment:
It doesn't seem like this test actually asserts anything differently. Is it worth the overhead of running it twice with a different parameter?

Reply:
I think it's worthwhile to ensure that the `failed_only` flag isn't completely broken.

Reply:
I thought about this some more and I talked myself out of it again. I think the test above checks that `failed_only` isn't completely broken, and if there's other behavior of the flag that's worth testing we should do that explicitly.